source/CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ if (UNIX AND NOT APPLE)
     set(SUPPORT_PRELOAD 1)
 endif()
 
+if(APPLE)
+    include(platform_apple.cmake)
+endif()
+
```

Comment:
Can we name the file CamelCase as CMake does things? (`PlatformApple.cmake`) Or specific to the content of the file, like `LibC++.cmake`?

Reply:
Done